lib/membrane/rtp/vad.ex

```diff
@@ -4,16 +4,9 @@ defmodule Membrane.RTP.VAD do
 
   To make this module work appropriate RTP header extension has to be set in SDP offer/answer.
 
-  This module is responsible for:
-    - receiving the RTP headers with audio levels in range [-127, 0] dBov
-    - mapping them to db (range [0, 127])
-    - calculating the epoch of the timestamp
-    - storing them in a queue
-    - asking the IsSpeakingEstimator if the audio levels indicate speech or silence
-    - emitting the VAD event if the estimation has changed
+  Sends `Membrane.RTP.VadEvent` when a score from `Membrane.RTP.VadUtils.IsSpeakingEstimator` changes.
 
-
-  The more detailed explanation of how the VAD algorithm can be found in the IsSpeakingEstimator module
+  A more detailed explanation of how the VAD algorithm can be found in the `Membrane.RTP.VadUtils.IsSpeakingEstimator` module
```

Comment:
The module name has been changed
```suggestion
  A more detailed explanation of how the VAD algorithm can be found in the `Membrane.RTP.Vad.IsSpeakingEstimator` module
```